src/components/smart/OeZoneerderZonePanel.vue

```diff
@@ -0,0 +1,442 @@
+<template>
+  <div ref="zonePanelRef" :class="{ closed: !panelOpen || !props.drawPanelEnabled }" class="panel">
+    <div ref="elementRef" class="zone-panel oe-ol-control ol-control ol-unselectable">
+      <button data-cy="zonePanelControl" @click="togglePanel"><font-awesome-icon :icon="['fas', 'bars']" /></button>
+    </div>
+
+    <vl-title class="panelHeader" tag-name="h4">
+      <font-awesome-icon class="pointer" :icon="['fas', 'bars']" @click="togglePanel" />
+      <span v-if="panelOpen" class="titleText">&nbsp;Zone samenstellen</span>
+    </vl-title>
+
+    <div v-if="panelOpen" class="panelBody">
+      <vl-input-group class="zone-input-group">
+        <template v-if="!addingWKT">
+          <vl-button
+            mod-narrow
+            :mod-secondary="!(activeDrawType === 'Polygon')"
+            title="teken polygoon"
+            @click="toggleDrawZone(true)"
+          >
+            <font-awesome-icon icon="draw-polygon" />
+          </vl-button>
+          <vl-button
+            mod-narrow
+            :mod-secondary="!(activeDrawType === 'Circle')"
+            title="teken cirkel"
+            @click="toggleDrawZone(true, 'Circle')"
+          >
+            <font-awesome-icon :icon="['far', 'circle']" />
+          </vl-button>
+          <vl-button
+            vl-button
+            mod-narrow
+            :mod-secondary="!selectPerceel"
+            title="selecteer perceel"
+            @click="startPerceelSelect()"
+          >
+            <font-awesome-icon icon="map-marker-alt" />
+          </vl-button>
+          <vl-button
+            data-cy="showWKTInput"
+            vl-button
+            mod-narrow
+            mod-secondary
+            title="WKT string"
+            @click="showWktInput()"
+            >WKT</vl-button
+          >
+        </template>
+        <template v-else>
+          <vl-input-field
+            data-cy="WKTInput"
+            id="map-address"
+            name="map-address"
+            placeholder="WKT string (Lambert72)"
+            mod-block
+            :model-value="WKTString"
+            @update:model-value="updateWKTString"
+          />
+          <vl-button data-cy="plaatsWKT" vl-button mod-narrow mod-secondary @click="drawWKTZone()">Plaats</vl-button>
+        </template>
+        <vl-button title="annuleren" vl-button mod-narrow mod-secondary @click="toggleDrawZone(false)">
+          <font-awesome-icon icon="cancel" />
+        </vl-button>
+      </vl-input-group>
+
+      <p><strong>Toegevoegde zones</strong></p>
```

Comment:
ik denk dat hier in de praktijk max 1 zone mag zijn, een multipolygon. zo zit locatie object in backend in elkaar, maar kan mis zijn
![image](https://github.com/OnroerendErfgoed/vue_component_library/assets/1849458/489732ef-be3e-45ed-9d53-707ac67f648c)

Reply:
Dit is bewust toegevoegd, indien de zone een multipolygon is wordt deze opgesplitst om gemakkeljiker te kunnen editeren, als je bv perongeluk een fout hebt gemaakt voor het opslaan

Reply:
ah nice